toolkits/global/packages/global-forms/README.md

```diff
@@ -80,10 +75,10 @@ The `hint` property adds hint text under the main label text but _inside_ the `<
 "hint": "This will be the email address you used when registering"
 ```
 
-The `optional` property adds _“(optional)”_ to the label text.
+To accompany the `required` property, there is `requiredSuffix`. You can use this to add an indicator after the label text, such as an asterisk. HTML is permitted, meaning you can hide the asterisk from assistive technologies using `aria-hidden`. The presence of the `required` attribute on the input itself suffices for screen reader software.
```

Comment:
As discussed on Slack, I think my preference would be to go back to the previous implementation where we have an `optional` property and have fields (implicitly) required by default.

This is because:

1. I hope it might discourage the collection of too much non-essential user data - which is general good practice
2. I think appending (optional) to a field's label is clearer than using an asterisk to denote a required field (though I don't have evidence for this and I accept that the latter is a common pattern).

After discussing with Head of Content Design - we think that appending (optional) to the field's label would be the clearest implementation.

Reply:
@amyhupe I removed `requiredSuffix` now, since this is redundant in a required-by-default approach.